### PR TITLE
feat(infrastructure): core resources

### PR DESCRIPTION
**Pull request by beejjacobs**
## Describe your changes

https://pins-ds.atlassian.net/browse/DEV-180

To be merged after #60 

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: infrastructure/local.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
+locals {
+  org              = "pins"
+  service_name     = "appeals-bo"
+  primary_location = "uksouth"
```
Comment (fabfab-kc) [inline infrastructure/local.tf:4]: Should this be `uk-south`?
https://github.com/claranet/terraform-azurerm-regions/blob/master/REGIONS.md

File: infrastructure/monitoring.tf [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  tags = local.tags
+}
+
+resource "azurerm_application_insights" "main" {
+  name                 = "${local.org}-ai-${local.resource_suffix}"
+  location             = module.primary_region.location
+  resource_group_name  = azurerm_resource_group.primary.name
+  workspace_id         = azurerm_log_analytics_workspace.main.id
+  application_type     = "other" # should this be Node.JS?
```
Comment (fabfab-kc) [inline infrastructure/monitoring.tf:16]: Maybe `web`?
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/application_insights

**Review by fabfab-kc — APPROVED** [review]
LGTM!

File: infrastructure/monitoring.tf (hunk 2 above)
Comment (beejjacobs) [reply infrastructure/monitoring.tf:16]: It says `web` is for `ASP.NET`, so I figured we either want general (other) or Node (Node.JS)?

File: infrastructure/local.tf (hunk 1 above)
Comment (beejjacobs) [reply infrastructure/local.tf:4]: Yes good spot - I'm sure I ran a plan and it didn't complain!

**Review by fabfab-kc — APPROVED** [review]
LGTM!

**Pull request merged**